logic/auth.js

```diff
@@ -149,6 +161,8 @@ async function login(user) {
         //unlock lnd wallet
         // await lndApiService.unlock(user.plainTextPassword, jwt);
 
+        deriveUmbrelSeed(user)
```

Comment:
Should we wrap this in an if condition to first check if the seed file doesn't already exist? Also, maybe it's a good idea to immediately create the seed file on register too, because it won't be created until the next login then.

Reply:
Yes, create on register sounds good, and yeah we can check for existence first.